City-of-Helsinki/drupal-helfi-platform-config
# UHF-2092: Changes to accordion

How to test:

1. `composer create-project City-of-Helsinki/drupal-helfi-platform:dev-main hel-platform --no-interaction --repository https://repository.drupal.hel.ninja/`
2. `cd hel-platform`
3. `composer require drupal/helfi_platform_config:dev-UHF-2092_changes_to_accordion && composer require drupal/hdbt:dev-UHF-2092_changes_to_accordion`
4. `make new`
5. Log in using the link provided by the install.
6. Go to add a basic page for example (any content with accordion paragraph will do)
7. When adding the accordion paragraph you should make sure that there is separate fields for Title, Title heading level and Description. These are new fields that can be filled or left blank. Add few accordions so that there is one with title, one with title and description and different header level than the default and one without either.
8. Save the page and make sure the accordion looks good. Refer to layouts here https://share.goabstract.com/3dd99d73-5b91-4915-bc3d-ba44c6bc592d?collectionLayerId=b1fe142e-99d6-4fc6-8df1-b208a33d775b&mode=design&sha=3ed1f997eb8f363abc02cc551daa225fe08a9309 and here https://share.goabstract.com/3dd99d73-5b91-4915-bc3d-ba44c6bc592d?collectionLayerId=e6baf1a7-8489-4a8b-bd4e-81e02d0ef477&mode=design&sha=53fde0e2b01e49c40f8dd4f4bc3722d591eb6913

Also check these:
https://github.com/City-of-Helsinki/drupal-hdbt/pull/159
https://github.com/City-of-Helsinki/drupal-helfi/pull/200
https://github.com/City-of-Helsinki/drupal-helfi-kymp/pull/124
https://github.com/City-of-Helsinki/drupal-helfi-sote/pull/153
https://github.com/City-of-Helsinki/drupal-helfi-strategia/pull/101
